network/src/main/java/org/openjdk/skara/network/RestRequest.java

```diff
@@ -328,10 +328,17 @@ private HttpResponse<String> sendRequest(HttpRequest.Builder request, boolean sk
         while (true) {
             try {
                 response = cache.send(authId, request, skipLimiter);
+                // If the status code is 301(Moved Permanently), follow the redirect link
+                if (response.statusCode() == 301) {
```

Comment:
I think we need to check the retryCount here like for the `401` case below, otherwise there is no break condition if the server keeps responding 301.

Reply:
My bad, will fix it